app/models/miq_server.rb

```diff
@@ -482,11 +482,7 @@ def restart
     _log.info("Server restart initiating...")
     self.update_attribute(:status, "restarting")
 
-    logfile = Rails.root.join("log/vmdb_restart.log")
-    File.delete(logfile) if File.exist?(logfile)
-
-    pid = spawn("nohup #{RESTART_SCRIPT}", [:out, :err] => [logfile, "a"])
```

Comment:
What's cool is that this gets rid of a Hakiri issue as well :gift:  (false positive, but still)